### PR TITLE
remove html

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: src/sunpy_sphinx_theme/__init__.py [hunk 1]
```diff
@@ -26,12 +26,12 @@ def default_navbar():
...
-                ("Presentations", "about/presentations.html", 2),
-                ("Community Roles", "about/roles.html", 2),
-                ("Meetings", "about/meetings.html", 2),
-                ("Code of Conduct", "coc.html", 2),
+                ("Our Mission", "about/index", 2),
+                ("SunPy Project", "about/project", 2),
+                ("Presentations", "about/presentations", 2),
+                ("Community Roles", "about/roles", 2),
+                ("Meetings", "about/meetings", 2),
+                ("Code of Conduct", "coc", 2),
```
Comment (nabobalis) [inline src/sunpy_sphinx_theme/__init__.py:34]:
```suggestion
                ("Our Mission", "about/index/", 2),
                ("SunPy Project", "about/project/", 2),
                ("Presentations", "about/presentations/", 2),
                ("Community Roles", "about/roles/", 2),
                ("Meetings", "about/meetings/", 2),
                ("Code of Conduct", "coc/", 2),
```

File: src/sunpy_sphinx_theme/__init__.py [hunk 2]
```diff
@@ -52,10 +52,10 @@ def default_navbar():
...
             ],
         ),
-        ("Affiliated Packages", "affiliated.html", 2),
-        ("Get Help", "help.html", 2),
-        ("Contribute", "contribute.html", 2),
-        ("Blog", "blog.html", 2),
+        ("Affiliated Packages", "affiliated", 2),
+        ("Get Help", "help", 2),
+        ("Contribute", "contribute", 2),
+        ("Blog", "blog", 2),
```
Comment (nabobalis) [inline src/sunpy_sphinx_theme/__init__.py:58]:
```suggestion
        ("Affiliated Packages", "affiliated/", 2),
        ("Get Help", "help/", 2),
        ("Contribute", "contribute/", 2),
        ("Blog", "blog/", 2),
```

**Review by Cadair — APPROVED** [review]

**Comment by nabobalis** [comment]
The eclipse example broke the CI here?!

File: src/sunpy_sphinx_theme/__init__.py [hunk 3]
```diff
@@ -26,12 +26,12 @@ def default_navbar():
         (
             "About",
             [
-                ("Our Mission", "about/index.html", 2),
-                ("SunPy Project", "about/project.html", 2),
-                ("Presentations", "about/presentations.html", 2),
-                ("Community Roles", "about/roles.html", 2),
-                ("Meetings", "about/meetings.html", 2),
-                ("Code of Conduct", "coc.html", 2),
+                ("Our Mission", "about/index/", 2),
```
Comment (nabobalis) [inline src/sunpy_sphinx_theme/__init__.py:29]:
```suggestion
                ("Our Mission", "about/index.html", 2),
```

File: src/sunpy_sphinx_theme/__init__.py [hunk 4]
```diff
@@ -27,11 +27,11 @@ def default_navbar():
             "About",
             [
                 ("Our Mission", "about/index.html", 2),
```
Comment (nabobalis) [inline src/sunpy_sphinx_theme/__init__.py:29]:
```suggestion
                ("Our Mission", "about/index/", 2),
```

File: src/sunpy_sphinx_theme/__init__.py (hunk 3 above)
Comment (nabobalis) [inline src/sunpy_sphinx_theme/__init__.py:29]:
```suggestion
                ("Our Mission", "about/", 2),
```

**Pull request merged**